### ISSUE TITLE
3D follow up 

### ISSUE THREAD

**Issue by cdsouthan** [OpenSourceMalaria/OSM_To_Do_List#500]
Last time at  BioITWorld  I chatted to sofware vendors who had varios 3D features with a view to them searching some OSM frontrunners.  I have to say, beyond polite interest,  traction was more or less zero. Also all seemed unwilling or unable to index PubChem as their search space.   I'm thus not that  motivated to bother again this year.  Assuming I might have a change of heart and even get the requisite spark of enthusiasm do we currently have the capacity to follow through?  (i.e.  for making some possibly difficult new "shapes" for testing)

**Comment by mattodd** [comment]
By "testing" you mean...?
Would be nice to see if shapes track potency, obv. Also with respect to the competition in #421

**Comment by cdsouthan** [comment]
What a coincidence ...

Jay Lakhan <jay@cresset-group.com>
3:54 PM (58 minutes ago)

to me 
Dear Christopher Southan,
 
You previously expressed an interest in Cresset’s technology, so I am emailing to provide you with an update on Cresset’s computational chemistry software. My name is Jay Lakhan and I am the Academic Liaison Officer at Cresset.
 
Cresset’s fields technology has advanced to describe the electrostatics and shape of both small molecules and proteins, producing a 3D field representation.  This has led to Cresset expanding its software from Ligand-based design tools to a Structure-based tool with the upcoming release of Flare.
Flare is now available for beta testing and will include:
Accurate and reliable protein preparation
Characterization of the protein active site, displaying protein interaction potentials calculated with the Cresset XED force field
Single thread and parallelized docking
Minimization of protein ligand complexes including active site residues and water in the moving shell
3D-RISM water analysis and WaterSwap analysis for investigating ligand and binding site energetics.
 
I would like to personally invite you to beta test the new tool. It would be a pleasure to discuss your thoughts on Cresset’s new and existing technology and how Cresset can support your research going forward.
 
I look forward to hearing from you soon.
 
Kind regards,
 
Jay
 
Jay Singh Lakhan
Academic Liaison Officer
Cresset
New Cambridge House, Bassingbourn Road, Litlington, Cambridgeshire, SG8 0SS, UK
tel: +44 (0)1223 858890  mobile: +44 (0)7482195135  fax: +44 (0)1223 853667
email: jay@cresset-group.com web: www.cresset-group.com

**Comment by drc007** [comment]
Do we have a high resolution X-ray structure with ligand bound?

**Comment by mattodd** [comment]
I'm still not clear what we're discussing here. @cdsouthan what's the idea, simply put? i.e. what inputs are you suggesting OSM provides, what outputs are you envisioning would result, and who would do the intermediate work? Are you implying volunteer companies/Cresset?
@drc007 if you mean Series 4/PfATP4 then no. We only have a target homology model. Is that what you mean?

**Comment by drc007** [comment]
As described above the Cresset technology requires a high resolution X-ray structure with ligand bound. I'm not convinced you would get anything useful from using a homology model.
@mattodd

**Comment by mattodd** [comment]
Well, we have data on a number of positives and negatives, and we now have several predictive models that have been submitted, so we could attempt to combine those with a homology model to look for e.g. overlapping binding sites. Yes, much less good than Xstal structure. But is this the kind of thing? i.e. what is the aim of all this? To develop a picture of protein-ligand interactions and hence help with future rational design? It feels like I missed a meeting - what's the aim here, simply put? Design, or just visualisation?

**Comment by drc007** [comment]
A typical use of the Cresset technology would be to do a virtual screen of 10M molecules and identify 10K potential hits. You then buy the 10K hits and screen them. Really impressive enrichment but is this what you want to do?
Alternatively if you have results from a binding assay where there is no concern about cell penetration, off-target activities etc. You can use to build a SAR model. I suspect you would need accurate binding affinities to be really useful.

**Comment by cdsouthan** [comment]
Minor apols for some confusion
1)  the objective was to get a few front runners run through a hypothesis-neutral 3D search by one of the small band of chemo software companies that profess to a) have new-ish capabilities in the area and b) who would give me the time of day at BioIT (even a drink from the back of the stand maybe...)
2) As it turned out one company did (sans booze unfortunately)  but shall remain nameless until they actually facilitate some kind of search result against various options of search space.
3) The communication from Cresset was a coincidental red herring in that they were not at BioIT to chat to (but we did have a friendly chat at GCC Fulda last year) and, as drc007 points out,  have a modus operandi that is not suited to a simple 3D search.

**Comment by holeung** [comment]
I have a license for Cresset Forge and can run some calculations for OSM.

**Comment by cdsouthan** [comment]
Sounds a top idea, should at least get some nice pictures.  Note if want comparative SAR sets I guess you can dig them out of the papers like rolling of a log but maybe I can help for the patents (even the MMV ones!)

**Comment by holeung** [comment]
I used Cresset Forge to predict 3D-SAR for the Series 4 compounds. I described this in my e-notebook. As pointed out above, without a crystal structure or high-confidence homology model, I'm not sure this will be very useful. But it's worth investigating.

**Comment by mattodd** [comment]
OK so I now see @holeung 's previous run is [here](http://malaria.ourexperiment.org/series_4_modeling/15729/Ligand_alignments.html). @cdsouthan is this what you had in mind?

**Comment by cdsouthan** [comment]
Thats good stuff but no,  I meant hypothesis-neutral 3D clustering across all OSM hits

**Comment by mattodd** [comment]
So if it's hypothesis-neutral you need only the [Master List](http://tinyurl.com/OSM-Compounds) and a target of interest?

**Comment by cdsouthan** [comment]
Actually "sans targets"  in first instance - just c what pharmacophore clusters emerge at what stringencies (e.g. via Cresset or anyone else's coloured sausages)

**Comment by holeung** [comment]
I'll take a poke.

**Comment by cdsouthan** [comment]
JFTR - who - on this project team has licences for the commercial heavyweight in silico stuff  e.g. Cresset,  OpenEye,  Schrodinger,  Molsoft (others) ?  

Also @holeung can u access the new fancy Cresset stuff they are tweeting about from the latest user meeting? or this needs more dosh I suppose....

**Comment by drc007** [comment]
Given this is the OPEN SOURCE malaria project, should we be using open source tools only?

**Comment by holeung** [comment]
OpenEye and ChemAxon offer free academic licenses. I have a paid license for Cresset Forge, which is their 3D-QSAR tool. I think you are referring to Cresset's new product, Flare. I played with it a bit in beta-testing but didn't feel drawn in. I think it's more useful for protein modeling and providing an integrated visual modeling system.

I try to use software that is open source or has a free academic license when possible. But in some areas, such as SAR and visualization, I don't think there are good open source alternatives. Much of the number-crunching type of commercial software can be implemented in free software with more effort and expertise.

I am considering getting a MOE license later this year. I would like to license Schrodinger, but their products are expensive.

**Comment by mcoster** [comment]
We have access to the Schrodinger suite through the Queensland Facility for Advanced Bioinformatics (QFAB), although I have no experience beyond Maestro-Macromodel-Jaguar. Is there anything in particular that could be done with this software that would be particularly useful for OSM?

**Comment by cdsouthan** [comment]
Can @holeung (or anyone else in team, maybe @drc007 ) knock out a few pics like this for the Series 4 paper?  (dont care what package OS or commercial - they just look cool) 

![capture](https://user-images.githubusercontent.com/3713939/28116167-30da1f14-6708-11e7-98c9-ee8868d35a71.JPG)

**Comment by drc007** [comment]
They look like Cresset representations?

**Comment by cdsouthan** [comment]
Yup, @drc007,  in fact from your very own blog  ...  http://www.cambridgemedchemconsulting.com/resources/hit_identification/virtual_screening.html

**Comment by holeung** [comment]
Sure. I'm happy to help with making 3D figures. Cresset has a free package, Torch Lite, for drawing and viewing.

[Cresset Torch Lite](http://www.cresset-group.com/products/torch/)
